sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.1 (Unreleased)
```

Comment:
Waiting to review and approve this PR until after we have decided what we are doing here.

Reply:
We've decided on 1.1, I'll revert the 3rd commit.